### ISSUE TITLE
[camelcase] definition for rule '@typescript-eslint/camelcase' was not found

### ISSUE THREAD

**Issue by ghaiklor** [typescript-eslint/typescript-eslint#2077]
**Repro**

When using `all` preset it throws "Definition for rule '@typescript-eslint/camelcase' was not found" error.

```JSON
  "extends": [
    "eslint:all",
    "plugin:@typescript-eslint/all",
    "plugin:import/typescript",
    "plugin:jest/all",
    "plugin:node/recommended",
    "plugin:promise/recommended",
    "standard-with-typescript"
  ],
```

**Expected Result**

It should not throw an error.

**Actual Result**

Getting errors:

```
error  Definition for rule '@typescript-eslint/camelcase' was not found  @typescript-eslint/camelcase
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `3.0.0` |
| `@typescript-eslint/parser`        | `3.0.0` |
| `TypeScript`                       | `3.9.3` |
| `ESLint`                           | `7.1.0` |
| `node`                             | `14.2.0` |
| `npm`                              | `6.14.4` |

**Comment by bradzacher** [comment]
Please take a look at the release notes for v3.
https://github.com/typescript-eslint/typescript-eslint/releases/tag/v3.0.0

**Comment by AnthonyLzq** [comment]
Same mistake here, this is mi `.eslintrc.json`:
```json
{
  "extends": [
    "eslint:recommended",
    "airbnb-typescript/base",
    "plugin:@typescript-eslint/recommended",
    "plugin:node/recommended",
    "plugin:prettier/recommended"
  ],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "project": "./tsconfig.json"
  },
  "plugins": ["@typescript-eslint", "sort-keys-fix"],
  "rules": {
    "@typescript-eslint/semi": ["error", "never"],
    "@typescript-eslint/naming-convention": [
      "error",
      {
        "selector": "default",
        "format": ["camelCase"]
      },
      {
        "selector": "variable",
        "format": ["camelCase", "UPPER_CASE"]
      },
      {
        "selector": "parameter",
        "format": ["camelCase"],
        "leadingUnderscore": "allow"
      },
      {
        "selector": "memberLike",
        "modifiers": ["private"],
        "format": ["camelCase"],
        "leadingUnderscore": "require"
      },
      {
        "selector": "typeLike",
        "format": ["PascalCase"]
      }
    ],
    "no-console": "off",
    "no-param-reassign": "off",
    "no-plusplus": ["error", { "allowForLoopAfterthoughts": true }],
    "radix": ["error", "as-needed"],
    "semi": "off",
    "sort-keys": "error"
  }
}
```

Versions:

| **package** | **version** |
|:-----------:|:-----------:|
| `@typescript-eslint/eslint-plugin` |  `3.0.0` |
| `@typescript-eslint/parser` | `3.0.0` |
| `eslint` | `7.1.0` |
| `eslint-config-airbnb-typescript` | `7.2.1` |
| `eslint-config-prettier`| `6.11.0` |
| `eslint-plugin-import` | `2.20.2` |
| `eslint-plugin-prettier` | `3.1.3` |
| `prettier` | `3.1.3` |
| `typescript` | `3.9.3` |

**Comment by bradzacher** [comment]
it's the same problem - `airbnb-typescript` has not been updated to support our v3 release.

**Comment by AnthonyLzq** [comment]
> it's the same problem - `airbnb-typescript` has not been updated to support our v3 release.

Oh, I didn't know that. I guess the only thing I can do is to wait for news.

**Comment by DimychOcean** [comment]
I added rules for resolve this problem:
- "@typescript-eslint/camelcase": "off"
- "@typescript-eslint/naming-convention": [...]

**Comment by r3faat1** [comment]
Getting the same issue... annoying issue. Above code did not fix it for me.

**Comment by bradzacher** [comment]
`eslint-config-standard-with-typescript` has been updated to support our v3 release.

`eslint-config-airbnb-typescript` has not yet - you can follow along at https://github.com/iamturns/eslint-config-airbnb-typescript/issues/95
- If you use this config, you must manually disable the `@typescript-eslint/camelcase` rule yourself in the meantime.
